.github/workflows/R-codecov-pkgmarkdown.yaml

```diff
@@ -0,0 +1,96 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
+# https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    branches:
+      - main
+      - master
+
+name: R-CMD-check
+
+jobs:
+  R-check:
+    runs-on: ubuntu-18.04
+    name: ubuntu-18.04 - R - ${{ matrix.config.r }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {r: 'release'}
+          - {r: 'oldrel'}
+          - {r: 'devel'}
+    env:
+      GITHUB_ACTION: true
+      R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      GH_TOKEN: ${{ secrets.TA_DA_ACCESS }}
```

Comment:
Do we want to delete this `secrets.TA_DA_ACCESS` business altogether because crunchy won't be updating tada?

Reply:
ah good catch. yes this needs to be removed